packages/client/src/v2-events/layouts/sidebar/Sidebar.tsx

```diff
@@ -49,14 +51,31 @@ export const Sidebar = ({
 }) => {
   const { slug: workqueueSlug } = useTypedParams(ROUTES.V2.WORKQUEUES.WORKQUEUE)
   const intl = useIntl()
-  const workqueues = useWorkqueueConfigurations()
+  const scopes = useSelector(getScope)
+
+  const { getOutbox } = useEvents()
+  const outbox = getOutbox()
+
+  const workqueuesWithoutOutboxOrDraft = useWorkqueueConfigurations()
+
+  const hasOutbox = hasOutboxWorkqueue(scopes ?? [])
+
+  const workqueues = [
+    ...(hasOutbox ? [WORKQUEUE_OUTBOX] : []),
+    ...workqueuesWithoutOutboxOrDraft
+  ]
```

Comment:
Why not just have outbox as a separate JSX element instead of merging it with the other workqueue data?